mmatongo/bookstore
# Bookstore: create and remove books

This PR adds the following changes and creates to the `dev-branch`


- Configure the Redux Store (`/src/redux/configureStore.js`) and importing the necessary methods from Redux:
  - Import `createStore`
  - Import `combineReducers`
  - Import `applyMiddleware` - To use some handy Redux middleware, like *redux logger*.
  - Import your `booksReducer`, which you are going to write in `/src/redux/books` directory.
- Write your book's actions and reducer (in `/src/redux/books/books.js`) using the [*ducks* pattern](https://github.com/erikras/ducks-modular-redux). 
- In your React component responsible for adding new books set the data inputs in the local React state (set *title* and *author*.
- In your React component responsible for removing books - implement that even by dispatching a corresponding action.
- Use the *redux-logger* to check if your application is working correctly - Open your browser's console and look for the logger messages. Upon every action dispatch, the logger will print:
  - **prev state** - the state of the whole Redux store before the current action was dispatched.
  - **action** - the action object that is being dispatched.
  - **next state** - the state of the Redux store after the current action was dispatched.
- Creates a `Store.js`
![src](https://user-images.githubusercontent.com/40806126/135148225-7991a55c-8eea-4765-a8e6-ebfb240d0e6b.png)

### NB A comment was accidentally deleted
